books.html

```diff
@@ -0,0 +1,2250 @@
+        
+
+        
+            <div class="product_price">
+                
+
+
+
+
+
+
+    
+        <p class="price_color">£47.82</p>
+    
+
+<p class="instock availability">
+    <i class="icon-ok"></i>
+    
+        In stock
+    
+</p>
+
+                
+                    
+
+
+
+
+
+
+    
+    <form>
+        <button type="submit" class="btn btn-primary btn-block" data-loading-text="Adding...">Add to basket</button>
+    </form>
+
+
+                
+            </div>
+        
+    </article>
+
+</li>
+                    
+                        <li class="col-xs-6 col-sm-4 col-md-3 col-lg-3">
+
+
+
+
+
+
+    <article class="product_pod">
+        
+            <div class="image_container">
+                
+                    
+                    <a href="../../sapiens-a-brief-history-of-humankind_996/index.html"><img src="../../../media/cache/be/a5/bea5697f2534a2f86a3ef27b5a8c12a6.jpg" alt="Sapiens: A Brief History of Humankind" class="thumbnail"></a>
+                    
+                
+            </div>
+        
+
+        
+            
+                <p class="star-rating Five">
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                </p>
+            
+        
+
+        
+            <h3><a href="../../sapiens-a-brief-history-of-humankind_996/index.html" title="Sapiens: A Brief History of Humankind">Sapiens: A Brief History ...</a></h3>
+        
+
+        
+            <div class="product_price">
+                
+
+
+
+
+
+
+    
+        <p class="price_color">£54.23</p>
+    
+
+<p class="instock availability">
+    <i class="icon-ok"></i>
+    
+        In stock
+    
+</p>
+
+                
+                    
+
+
+
+
+
+
+    
+    <form>
+        <button type="submit" class="btn btn-primary btn-block" data-loading-text="Adding...">Add to basket</button>
+    </form>
+
+
+                
+            </div>
+        
+    </article>
+
+</li>
+                    
+                        <li class="col-xs-6 col-sm-4 col-md-3 col-lg-3">
+
+
+
+
+
+
+    <article class="product_pod">
+        
+            <div class="image_container">
+                
+                    
+                    <a href="../../the-requiem-red_995/index.html"><img src="../../../media/cache/68/33/68339b4c9bc034267e1da611ab3b34f8.jpg" alt="The Requiem Red" class="thumbnail"></a>
+                    
+                
+            </div>
+        
+
+        
+            
+                <p class="star-rating One">
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
+                    <i class="icon-star"></i>
```

Comment:
The star-rating class 'One' does not match the number of filled stars (five). This inconsistency might confuse users or affect the visual representation of ratings.